LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Created by Bluedot Innovation in 2023.
+Copyright © 2023 Bluedot Innovation. All rights reserved.
+
+By downloading or using the iOS Bluedot Point SDK for Xamarin, You agree to the Bluedot Terms and Conditions (http://www.bluedotinnovation.com/html/downloads/pdfs/terms-and-conditions-bluedot-070814.pdf)
+and Privacy Policy (http://www.bluedotinnovation.com/html/downloads/pdfs/privacy-policy-bluedot-170815.pdf)
+and Billing Policy(http://www.bluedotinnovation.com/html/downloads/pdfs/privacy-policy-bluedot-170815.pdf)
```

Comment:
This link doesn't work. Can you update it with a working one from Dan?

Reply:
Just updated the urls